ComSemApp/templates/ComSemApp/student/assessment_results.html

```diff
@@ -3,6 +3,15 @@
 {% load static %}
 
 {% block content %}
+<style>
```

Comment:
Hmmm... Is this where stylesheet elements should be defined?

Reply:
I am going to keep it at the top for right now. I couldn't find much info about convention of where it should be put but it doesn't seem to be too important where it is as far as I know